ArjunaJTA/jta/pom.xml

```diff
@@ -462,5 +466,20 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+        <id>jdk17</id>
+        <activation>
+            <jdk>[17,)</jdk>
+            <property>
+              <name>arqProfileActivated</name>
+              <value>true</value>
+            </property>
+        </activation>
+        <properties>
+            <jvm.args.modular>
+                --add-opens=java.base/java.util=ALL-UNNAMED
```

Comment:
@mmusgrov FYI I lost your question somewhere but this is essentially only needed for the tests we ran in Arquillian and not the main ones so I reduced when it was activiated. I am going to put a comment though in the pom.xml